bifrost/app/blog/page.tsx

```diff
@@ -190,6 +190,11 @@ export type BlogStructure =
     };
 
 const blogContent: BlogStructure[] = [
+  {
+    dynmaicEntry: {
+      folderName: "prompt-engineering-tools",
+    },
+  },
```

Comment:
NGL whoever made it this easy deserves a reward

Reply:
i agree they deserve a raise